### PR TITLE
refact(jiva): restrict recurring ReadOnly logs by adding flag

### PR DESCRIPTION
**Pull request by utkarshmani1997**
Signed-off-by: Utkarsh Mani Tripathi <utkarsh.tripathi@mayadata.io>

### CODE REVIEW

File: controller/control.go [hunk 1]
```diff
@@ -966,12 +969,14 @@ func (c *Controller) WriteAt(b []byte, off int64) (int, error) {
...
 	c.Lock()
-	if c.ReadOnly == true {
+	if c.ReadOnly && !c.logged {
+		c.logged = true
 		err := fmt.Errorf("Mode: ReadOnly")
 		c.Unlock()
 		time.Sleep(1 * time.Second)
 		return -1, err
 	}
+	c.logged = false
```
Comment (payes) [inline controller/control.go]: Let's also put a log saying, volume started serving IOs. Even though we have the log times when the volume became RO and RW, but we would also like to know the timestamps when the first time IOs were unserved and when the first time IOs were being served again.

Comment (payes) [inline controller/control.go]: Let's also put a log saying, volume started serving IOs. Even though we have the log times when the volume became RO and RW, but we would also like to know the timestamps when the first time IOs were unserved and when the first time IOs were being served again.

File: controller/control.go [hunk 2]
```diff
@@ -935,12 +936,14 @@ func (c *Controller) Start(addresses ...string) error {
 // on the app.
 func (c *Controller) WriteAt(b []byte, off int64) (int, error) {
 	c.Lock()
-	if c.ReadOnly == true {
+	if c.ReadOnly && !c.logged {
```
Comment (payes) [inline controller/control.go]: We shouldn't skip this entire block, only logging needs to be disabled.

Comment (utkarshmani1997) [reply controller/control.go]: good catch

**Comment by utkarshmani1997** [comment]
closing this PR, as this is not solving the problem

**Pull request closed without merging**